content/operate/rs/databases/import-export/flush.md

```diff
@@ -41,6 +42,45 @@ redis-cli -h redis-12345.cluster.local -p 9443 -a xyz flushall
 Port 9443 is the default [port configuration]({{< relref "/operate/rs/networking/port-configurations#https://docs.redis.com/latest/rs/networking/port-configurations#ports-and-port-ranges-used-by-redis-enterprise-software" >}}).
 {{< /note >}}
 
+## Flush data from an OSS Cluster API database
+
+When using the OSS Cluster API, the `FLUSHDB` command only flushes the keys of the shard you're connected to, not all keys in the database. To flush all data from an OSS Cluster API database, you need to connect to each endpoint IP address and execute the flush command.
+
+Follow these steps to flush all data from an OSS Cluster API database:
+
+1. Find all IP addresses associated with the database endpoint using the `dig` command:
+
+   ```sh
+   dig +noall +answer <database-endpoint>
+   ```
+
+2. Connect to each IP address using redis-cli and execute the `FLUSHDB` command:
+
+   ```sh
+   redis-cli -h <ip-address> -p <port> flushdb
+   ```
+
+### Example
+
+1. Find all IP addresses for the database endpoint:
+
+   ```sh
+   $ dig +noall +answer redis-19674.test2f4e15b0.cs.redislabs.com
+   redis-19674.test2f4e15b0.cs.redislabs.com. 5 IN A 52.200.243.146
+   redis-19674.test2f4e15b0.cs.redislabs.com. 5 IN A 54.235.52.171
```

Comment:
Maybe change the IP address examples to the ones reserved for documentation use.
```suggestion
   $ dig +noall +answer redis-19674.test2f4e15b0.cs.redislabs.com
   redis-19674.test2f4e15b0.cs.redislabs.com. 5 IN A 192.0.2.0
   redis-19674.test2f4e15b0.cs.redislabs.com. 5 IN A 198.51.100.0
```